libs/api/repository/src/lib/gn4/gn4-repository.ts

```diff
@@ -323,6 +333,10 @@ export class Gn4Repository implements RecordsRepositoryInterface {
     )
   }
 
+  draftIsTemporary(uniqueIdentifier: string): boolean {
+    return uniqueIdentifier.startsWith('TEMP-ID-')
+  }
```

Comment:
@jahow I tried to do a proper check here, with loading the record XML, but there seems to be a retry mecanism, ending up with multiple requests, each taking more and more time.
Do you have any clue about this? Or an idea to make this check more robust?